README.rst

```diff
@@ -20,33 +20,85 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Usage
+=====
+
+PGShovel is a tool which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL data out of the database and into an ordered stream of transaction-aware mutation messages.  These messages can then be consumed by 3rd party systems as needed.  Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
+
+At Disqus PGShovel is actively being developed to replicate changes from our master Postgres database - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to use the mutation stream to power a denormalized objects database, offering highly available low-latency access to object data.
+
 Concepts
 ========
 
+PGShovel has a few core concepts which are important to understanding it:
+
 Cluster
 -------
 
-A cluster contains replication sets.
+A Cluster contains a collection of Replication Sets.
 
 Replication Set
 ---------------
 
-Replication sets contain a collection of tables that are replicated together.
+Replication Sets contain configuration defining a collection of tables that are replicated together.
+
+Batch Operations
```

Comment:
I think this seems a bit low level for a core concept. I do think it needs to be documented, but not sure this level of detail makes sense in the overview.